Shared/Features/Freaks/View/MultipleSelectionRow.swift

```diff
@@ -0,0 +1,23 @@
+import SwiftUI
+
+struct MultipleSelectionRow: View {
+    @StateObject var viewModel = FreaksViewModel()
```

Comment:
We are still creating a new instance of the FreaksViewModel inside a child view. I think the better approach is to create the @StateObject in the parent view (Freaks screen) and pass down the reference to the child views as an @ObservedObject

Reply:
Got it. It should be fine now